ui/helpers/jobHelper.js

```diff
@@ -14,6 +14,10 @@ const btnClasses = {
   'in progress': "btn-dkgray",
 };
 
+export const allResultStates = ['success', 'testfailed', 'busted', 'exception', 'retry', 'usercancel', 'running', 'pending', 'superseded', 'runnable'];
```

Comment:
There's a `thAllResultStates` in `constants.js` already?

Reply:
I blame this on temporary insanity...  :)  Thanks for catching my mistake.

Reply:
We by necessity have a lot of constants - it's hard to keep track! :-)